src/kpatch_user.c

```diff
@@ -954,13 +988,23 @@ int main(int argc, char *argv[])
 {
 	int opt;
 
-	while ((opt = getopt(argc, argv, "+vh")) != EOF) {
+	while ((opt = getopt(argc, argv, "+vl:h")) != EOF) {
 		switch (opt) {
 			case 'v':
 				log_level += 1;
 				break;
 			case 'h':
 				return usage(NULL);
+			case 'l':
+#ifdef STRESS_TEST
+				if (strlen(optarg) + 9 >= LOG_NAME_LEN) { // 9 - max suffix length
+					usage_stresstest();
+					kpfatal("Can't initialize log\n");
+					break;
+				}
+				strncpy(test_log_base, optarg, LOG_NAME_LEN);
```

Comment:
Is it possible to just use `strncpy` and then check if last byte is zero? (this requires `test_log_base` to be declared static).